_sass/components/_list.scss

```diff
@@ -66,9 +66,9 @@ ul.help-question-list {
   p {
     // TODO: Remove once `$theme-body-line-height` is assigned as consistent with `$line-height`.
     position: relative;
-    $line-height-offset: round(
-        ($line-height - line-height($theme-body-font-family, $theme-body-line-height)) * 100
-      ) / 100 / -2;
+    $line-height-offset: -0.02 * round(
+      ($line-height - line-height($theme-body-font-family, $theme-body-line-height)) * 100
+    );
```

Comment:
The math flattening is hilarious to me. It's because `/` is overloaded in some versions of SASS right?

maybe for clarity could we split it into multiple `*`?

```suggestion
    $line-height-offset: -2 * 0.01 * round(
      100 * ($line-height - line-height($theme-body-font-family, $theme-body-line-height))
    );
```

Reply:
Yeah, it's due to the ["Slash as Division" deprecation](https://sass-lang.com/documentation/breaking-changes/slash-div), which started failing once switching to Dart Sass.

I'm open to splitting it out if it helps keep it readable; to be honest, I don't recall what this style is doing or why these numbers are chosen, so to me they're confusing any way they're arranged 🤷

Reply:
There's a round to 2 decimals that I can see `(0.01 * round(100 * x)` and the divide by -2... no idea

Reply:
Now that you've explained it I see that my equivalent logic is incorrect, since `X / 100 / -2` should be `X * -0.005`, not `X * -0.02`. Let me see what this logic is all about and whether I can remove it or simplify it, otherwise I'll do `X * 0.01 * -0.5` (or maybe just pull in `sass:math` and do some `math.div` to keep it more like how it was originally implemented).